.github/kokoro/run.sh

```diff
@@ -36,9 +36,26 @@ echo "----------------------------------------"
 	hash -r
 	conda activate sv-test-env
 	set -x
-        make generate-tests
-	make tests USE_ALL_RUNNERS=1 -j$CORES
-        make report USE_ALL_RUNNERS=1
+	make generate-tests
+
+	set -x +e
+	tmp=`mktemp`
+	script --return --flush --command "make USE_ALL_RUNNERS=1 -j$CORES --keep-going tests" $tmp
+	TESTS_RET=$?
+	set +x -e
+
+	if [[ $TESTS_RET != 0 ]]; then
+		echo "----------------------------------------"
+		echo "A failure occurred during test running."
+		echo "----------------------------------------"
+		make USE_ALL_RUNNERS=1 -j tests
```

Comment:
with `-j`, wouldn't make fork as many as possible ? We should probably limit to `-j1`.

Reply:
Fixed.